sigopt/factory.py

```diff
@@ -48,6 +51,18 @@ def _on_experiment_created(self, experiment):
       experiment.id,
     )
 
+  def create_project(self, project_name):
+    client_id = get_client_id(self.connection)
+    try:
+      project = self.connection.clients(client_id).projects().create(id=self.project, name=project_name)
+    except ApiException as e:
+      if e.status_code == http.HTTPStatus.CONFLICT:
+        raise ConflictingProjectException(self.project)
```

Comment:
Connected to above comment, will this still be `None`, even if the backend converted to some safe repr? If so, this error message will be misleading